inference-engine/thirdparty/clDNN/src/network.cpp

```diff
@@ -498,6 +502,11 @@ void network_impl::execute(const std::vector<event::ptr>& events) {
         }
 #endif
 #endif
+        CLDNN_DEBUG_CONFIG_IF(debug_config->verbose >= 1) {
```

Comment:
maybe shorter name would be more handy? Like `DEBUG_IF` or `GPU_DEBUG_IF`?

Reply:
Or maybe we should create helper class like `DebugLogger` with set of helper methods which will hide all these debug level checks?

Reply:
The main purpose of `CONFIG_IF` is not log level checking, but to help compilers to eliminate this conditional statement. I think it would be good to have additional logging, but not now..